src/server/worker.js

```diff
@@ -64,7 +64,9 @@ export function run(worker) {
   }), graphQLHandler);
 
   // server-side rendering for emails
-  app.get('/email', emailSSR);
+  if (!PROD) {
```

Comment:
I decided to go simple here. Why remove the code when it might be useful to @ackernaut et al in the future? We all good with this?